sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuFileFormatDataWriter.scala

```diff
@@ -963,7 +967,7 @@ class GpuDynamicPartitionDataConcurrentWriter(
     withResource(t) { _ =>
       val batch = GpuColumnVector.from(t, outDataTypes)
       if (!needSplitBatch(maxRecordsPerFile, status.writerStatus.recordsInFile, batch.numRows())) {
-        statsTrackers.foreach(_.newBatch(batch))
+        statsTrackers.foreach(_.newBatch(status.writerStatus.outputWriter.path(), batch))
```

Comment:
nit: why have `()` on some calls to path, but not all of them?

Reply:
That's not intentional.  They should all have parentheses since the method definition has it.